proposal.md

```diff
@@ -0,0 +1,8 @@
+## Project Proposal Description
```

Comment:
move to `projects/python/<whatever you want to call it right now>.md`

Reply:
updated

Reply:
> move to `projects/python/<whatever you want to call it right now>.md`

updated